data/hails.txt

```diff
@@ -2030,7 +2030,288 @@ phrase "friendly author"
+		"Bootless"
+		"Churlish"
+		"Cockered"
+		"Clouted"
+		"Craven"
+		"Currish"
+		"Dankish"
+		"Dissembling"
+		"Droning"
+		"Errant"
+		"Fawning"
+		"Fobbing"
+		"Froward"
+		"Frothy"
+		"Gleeking"
+		"Goatish"
+		"Gorbellied"
+		"Impertinent"
+		"Infectious"
+		"Jarring"
+		"Loggerheaded"
+		"Lumpish"
+		"Mammering"
+		"Mangled"
+		"Mewling"
+		"Paunchy"
+		"Pribbling"
+		"Puking"
+		"Puny"
+		"Qualling"
+		"Rank"
+		"Reeky"
+		"Roguish"
+		"Ruttish"
+		"Saucy"
+		"Spleeny"
+		"Spongy"
+		"Surly"
+		"Tottering"
+		"Unmuzzled"
+		"Vain"
+		"Venomed"
+		"Villainous"
+		"Warped"
+		"Wayward"
+		"Weedy"
+		"Yeasty"
+		"Some"
+		"Some"
+		"Some"
+		"Some"
+		"Some"
+		"Some"
+		"Some"
+		"Some"
+		"Those"
+		"Those"
+		"Those"
+		"Those"
+		"Those"
+		"Those"
+		"Those"
+		"Those"
+	word
+		" "
+	word
+		"base-court"
+		"bat-fowling"
+		"beef-witted"
+		"beetle-headed"
+		"boil-brained"
+		"clapper-clawed"
+		"clay-brained"
+		"common-kissing"
+		"crook-pated"
+		"dismal-dreaming"
+		"dizzy-eyed"
+		"doghearted"
+		"dread-bolted"
+		"earth-vexing"
+		"elf-skinned"
+		"fat-kidneyed"
+		"fen-sucked"
+		"flap-mouthed"
+		"fly-bitten"
+		"folly-fallen"
+		"fool-born"
+		"full-gorged"
+		"guts-griping"
+		"half-faced"
+		"hasty-witted"
+		"hedge-born"
+		"hell-hated"
+		"idle-headed"
+		"ill-breeding"
+		"ill-nurtured"
+		"knotty-pated"
+		"milk-livered"
+		"motley-minded"
+		"onion-eyed"
+		"plume-plucked"
+		"pottle-deep"
+		"pox-marked"
+		"reeling-ripe"
+		"rough-hewn"
+		"rude-growing"
+		"rump-fed"
+		"shard-borne"
+		"sheep-biting"
+		"spur-galled"
+		"swag-bellied"
+		"tardy-gaited"
+		"tickle-brained"
+		"toad-spotted"
+		"unchin-snouted"
+		"weather-bitten"
+	
+	word
+		" "
+	word
+		"apple-johns"
+		"baggages"
+		"barnacles"
+		"bladders"
+		"boar-pigs"
+		"bugbears"
+		"bum-baileys"
+		"canker-blossoms"
+		"clack-dishes"
+		"clotpoles"
+		"coxcombs"
+		"codpieces"
+		"death-tokens"
+		"dewberries"
+		"flap-dragons"
+		"flax-wenches"
+		"flirt-gills"
+		"foot-lickers"
+		"fustilarians"
+		"giglets"
+		"gudgeons"
+		"haggards"
+		"harpies"
+		"hedge-pigs"
+		"horn-beasts"
+		"hugger-muggers"
+		"joitheads"
+		"lewdsters"
+		"louts"
+		"maggot-pies"
+		"malt-worms"
+		"mammets"
+		"measles"
+		"minnows"
+		"miscreants"
+		"moldwarps"
+		"mumble-newss"
+		"nut-hooks"
+		"pigeon-eggs"
+		"pignuts"
+		"puttocks"
+		"pumpions"
+		"ratsbanes"
+		"scuts"
+		"skainsmates"
+		"strumpets"
+		"varlots"
+		"vassals"
+		"whey-faces"
+		"wagtails"
+	word
+		" "
+	word
+		"wrecked our ship!"
+		"left us drifting in space!"
+		"got the jump on us!"
+		"left us here to die!"
+		"disabled us!"
+		"fried most of our ship functions!"
+		"suddenly came onto us!"
+		"bested us!"
+		"thought they could simply leave us here!"
+	word
+		" "
+	word
+		"We could use any help we can get!"
+		"Could you lend a hand?"
+		"We can't seem to repair it ourselves, could you help?"
+		"We need help getting our ship running again!"
+		"Can you help us fix it please?"
```

Comment:
another please that [should? could?] be comma'd
the next one shouldn't, in case you wonder, because please is before the rest of the sentence